.github/workflows/cd.yml

```diff
@@ -0,0 +1,24 @@
+name: cd
```

Comment:
Were you able to test this locally?

Reply:
Maintainer Has to set it up. Right now, it is not working. Releasing is set to Manual right now.